x-pack/plugins/cloud_security_posture/public/test/mock_server/handlers/api/fleet_epm_packages_cloud_security_posture.ts

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
should we follow the approach from `internal` and have `internal/fleet` and `internal/licensing`? And another thing - is there a benefit in reflecting the path structure exactly? Wouldn't `internal` and `external` make more sense? or if the internal vs external doesn't make a difference in mocking, just skip this level and just have `fleet`, `cloud_security_posture`, `license` on the top level of `handlers`? Having this structure which kind of mimic the path but not exactly confused my until I looked deeper in the code

Reply:
The path convention is up to us, so your suggestion for simplicity makes a lot of sense, I covered that in [Option 2](https://github.com/elastic/kibana/pull/184555#discussion_r1623006022) here